ci/prow/config.yaml

```diff
@@ -5618,6 +5618,78 @@ periodics:
     - name: housekeeping-github-token
       secret:
         secretName: housekeeping-github-token
+- cron: "30 07 * * *"
+  name: ci-knative-serving-recreate-clusters
+  agent: kubernetes
+  decorate: true
+  extra_refs:
+  - org: knative
+    repo: serving
+    base_ref: master
+    path_alias: knative.dev/serving
+  spec:
+    containers:
+    - image: gcr.io/knative-tests/test-infra/prow-tests:stable
+      imagePullPolicy: Always
+      command:
+      - runner.sh
+      args:
+      - "./test/performance/tools/recreate_clusters.sh"
+      volumeMounts:
+      - name: test-account
+        mountPath: /etc/test-account
+        readOnly: true
+      - name: performance-test
+        mountPath: /etc/performance-test
+        readOnly: true
+      env:
+      - name: GOOGLE_APPLICATION_CREDENTIALS
+        value: /etc/test-account/service-account.json
+      - name: PERF_TEST_GOOGLE_APPLICATION_CREDENTIALS
+        value: /etc/performance-test/service-account.json
+    volumes:
+    - name: test-account
+      secret:
+        secretName: test-account
+    - name: performance-test
+      secret:
+        secretName: performance-test
+- cron: "0 * * * *"
+  name: ci-knative-serving-update-clusters
+  agent: kubernetes
+  decorate: true
+  extra_refs:
+  - org: knative
+    repo: serving
+    base_ref: master
+    path_alias: knative.dev/serving
+  spec:
+    containers:
+    - image: gcr.io/knative-tests/test-infra/prow-tests:stable
+      imagePullPolicy: Always
+      command:
+      - runner.sh
+      args:
+      - "./test/performance/tools/update_clusters.sh"
+      volumeMounts:
+      - name: test-account
+        mountPath: /etc/test-account
+        readOnly: true
+      - name: performance-test
+        mountPath: /etc/performance-test
+        readOnly: true
+      env:
+      - name: GOOGLE_APPLICATION_CREDENTIALS
+        value: /etc/test-account/service-account.json
+      - name: PERF_TEST_GOOGLE_APPLICATION_CREDENTIALS
+        value: /etc/performance-test/service-account.json
+    volumes:
+    - name: test-account
+      secret:
+        secretName: test-account
+    - name: performance-test
```

Comment:
this secret is in the default ns. This needs to be in test-pods ns

Reply:
Thanks for catching this! I realized it but forgot to make the update. 
Added.